dataactcore/scripts/pipeline/get_fsrs_updates.py

```diff
@@ -64,72 +141,8 @@ def main():
 
     metrics_json['start_date'] = mod_date
 
-    logger.info("Starting SQL query of financial assistance records from {} to present...".format(mod_date))
     sess = GlobalDB.db().session
```

Comment:
Since `sess` is only used for the SQL call, does it make sense to make the sess in the function and save on a parameter?